helm/flink-operator/crds/flinkdeployments.flink.apache.org-v1.yml

```diff
@@ -26,6 +26,22 @@ spec:
               serviceAccount:
                 type: string
               flinkVersion:
+                enum:
+                - v1_3
```

Comment:
Do we really want those old (irrelevant) Flink versions here? It might be better to only list the versions that are actually supported. That probably means that we need to have our own enum. And that should be fine, since the validation can still check it is one of `FlinkVersion`. That would also avoid having to copy the entire `FlinkVersion` class, we can add that check when 1.15 becomes the new minimum version.

Reply:
I don't have a problem with having our own enum either (that way we would have immediate validation for the accepted versions). @wangyang0918 , @morhidi what do you think?

Reply:
makes perfect sense to me

Reply:
Having our own enum of FlinkVersion also makes sense to me.